teams/teams-ps/teams/Get-TeamTargetingHierarchyStatus.md

```diff
@@ -0,0 +1,89 @@
+---
+external help file: Microsoft.Open.Teams.CommonLibrary.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/connect-microsoftteams
+schema: 2.0.0
+author: mbergamo
+ms.author: mbergamo
+ms.reviewer:
+---
+# Get-TeamTargetingHierarchyStatus
+
+## SYNOPSIS
+
+Get the status of a hierarchy upload (see [Set-TeamTargetingHierarchy](https://docs.microsoft.com/powershell/module/teams/get-teamtargetinghierarchy))
```

Comment:
Can you move the reference to `Set-TeamTargetingHierarchy` to the Related Links section?